plugins/orchestrator-backend/package.json

```diff
@@ -36,21 +36,21 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.19.4",
-    "@backstage/backend-plugin-api": "^0.6.2",
-    "@backstage/catalog-client": "^1.4.3",
-    "@backstage/config": "^1.0.8",
-    "@backstage/core-plugin-api": "^1.5.3",
+    "@backstage/backend-common": "^0.19.5",
+    "@backstage/backend-plugin-api": "^0.6.3",
+    "@backstage/catalog-client": "^1.4.4",
+    "@backstage/config": "^1.1.0",
+    "@backstage/core-plugin-api": "^1.6.0",
     "@backstage/integration": "^1.7.0",
-    "@backstage/plugin-catalog-node": "^1.4.0",
-    "@backstage/plugin-events-backend": "^0.2.8",
-    "@backstage/plugin-events-node": "^0.2.8",
+    "@backstage/plugin-catalog-node": "^1.4.4",
+    "@backstage/plugin-events-backend": "^0.2.12",
+    "@backstage/plugin-events-node": "^0.2.12",
     "@backstage/plugin-scaffolder-backend": "^1.17.0",
-    "@backstage/plugin-scaffolder-node": "^0.2.2",
-    "@backstage/types": "^1.1.0",
+    "@backstage/plugin-scaffolder-node": "^0.2.3",
+    "@backstage/types": "^1.1.1",
     "@janus-idp/backstage-plugin-orchestrator-common": "0.0.1",
-    "@octokit/rest": "^19.0.3",
     "@severlessworkflow/sdk-typescript": "^3.0.3",
+    "octokit": "^2.0.0",
```

Comment:
I believe we need to keep `"@octokit/rest": "^19.0.3",` and remove `"octokit": "^2.0.0",`.

Reply:
I'll remove octokit from deps, in the end we are not using it